.pipelines/cni/pipeline.yaml

```
@@ -107,16 +107,16 @@ stages:
       clusterType: overlay-byocni-nokubeproxy-up
       clusterName: "cilium-overlay"
       nodeCount: ${CILIUM_NODE_COUNT}
-      vmSize: ${CILIUM_VM_SKU}
+      vmSize: ${CILIUM_VM_SIZE}

```

```suggestion
      vmSize: ${VM_SIZE_CILIUM}
```
it should follow the least-to-most-specific ordering convention so that the lexicographic sort will group variants of the same arg together